kie-wb-smoke-tests/src/main/java/org/kie/smoke/wb/selenium/model/WebDriverFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.smoke.wb.selenium.model;
+
+import java.io.File;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+
+/**
+ * Class responsible for instantiating WebDriver instance.
+ */
+public class WebDriverFactory {
+
+    // IEDriverServer binary downloaded by antrun plugin in pom.xml
+    private static final File IE_DRIVER_BINARY = new File(System.getProperty("java.io.tmpdir"), "IEDriverServer.exe");
```

Comment:
Although the filename won't likely change, I agree that it would be better to pass property to the test via failsafe plugin configuration.